.github/dependabot.yml

```diff
@@ -4,20 +4,20 @@ updates:
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: "weekly"
+      interval: "monthly"
     commit-message:
-      prefix: "chore(github-actions dependencies):"
+      prefix: "chore(dependencies): GITHUB-ACTIONS"
```

Comment:
Will this fix #456 ?

Reply:
I think it won't fit the rules either, as the format is sentence-case (it would work with e.g. "GitHub Actions"

Reply:
It works well :) 

```
❯ echo "chore(dependencies): GITHUB-ACTIONS" | npx commitlint --verbose
⧗   input: chore(dependencies): GITHUB-ACTIONS
✔   found 0 problems, 0 warnings
```

```
❯ echo "chore(dependencies): gITHUB-ACTIONS" | npx commitlint --verbose
⧗   input: chore(dependencies): gITHUB-ACTIONS
✖   subject must be sentence-case [subject-case]

✖   found 1 problems, 0 warnings
```